src/states/node.rs

```diff
@@ -3342,36 +3342,33 @@ impl Bootstrapped for Node {
             }
             return Ok(());
         }
-        if !self.add_to_pending_acks(&routing_msg, route, expires_at) {
+
+        use crate::routing_table::Authority::*;
+        let sending_sec = if route == 0 {
+            match routing_msg.src {
+                ClientManager(_) | NaeManager(_) | NodeManager(_) | ManagedNode(_) | Section(_)
+                | PrefixSection(_)
```

Comment:
I think the idea with `PrefixSection` was that the node collecting the signatures could be in our sibling section. If we put in a different source section, our signature won't match. Even with the "N/3→N/3" change, I think that might still be relevant?

Reply:
(As discussed, this is probably not relevant anymore. Possibly error if source is `PrefixSection`.)